cmd/create/cluster/cmd.go

```diff
@@ -2209,6 +2219,52 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	// Audit Log Forwarding
+	auditLogRoleARN := args.AuditLogRoleARN
+
+	if auditLogRoleARN != "" && !isHostedCP {
```

Comment:
NP
I wonder if we can check that earlier, to give faster feedback.

Reply:
Not sure where the best area / placement should be here :slightly_frowning_face: 
I personally thought it was best to group this check alongside other code statements related to audit log forwarding.